### DISCUSSION TITLE
Can this code be removed?

### DISCUSSION THREAD

**Discussion by mowangjuanzi** [laravel/framework#41083] [Q&A] [answered]
file path: https://github.com/laravel/framework/blob/9.x/src/Illuminate/Pagination/resources/views/bootstrap-5.blade.php

```html
<div>
                <p class="small text-muted">
                    {!! __('Showing') !!}
                    <span class="font-medium">{{ $paginator->firstItem() }}</span>
                    {!! __('to') !!}
                    <span class="font-medium">{{ $paginator->lastItem() }}</span>
                    {!! __('of') !!}
                    <span class="font-medium">{{ $paginator->total() }}</span>
                    {!! __('results') !!}
                </p>
            </div>
```
Can this code be removed?

Because the previous bootstrap 4 did not have the following content：

![无标题](https://user-images.githubusercontent.com/13846040/154518955-3640cce1-fc50-4492-aa93-32f25efdf4a3.png)

**Comment by gguney** [comment, marked as answer]
1. First solution can be publishing vendor views then removing that part of the view.
- Run `php artisan vendor:publish`
- Enter the number left side of the `Provider: Illuminate\Pagination\PaginationServiceProvider`
- After that you can see `bootstrap-5.blade.php` under resources/views/vendor/pagination
- Then edit the views as you like.

2. You can add simple css like 
`.text-muted {
    display: none;
}` 

Or you may add selector from this div's parent element.
However, it may effect other elements that has "text-muted"

**Reply by mowangjuanzi** [reply]
yes. At present, it seems that this is the only way
